### ISSUE TITLE
Getting errors in build in Ubuntu 20.04

### ISSUE THREAD

**Issue by sharad1126** [RangeNetworks/dev#94]
I was getting errors on installing the `python-software-properties` but on stackoverflow and all, it says to only install `software-properties-common`. I did the same and then commented out this in `build.sh`. But now when I am trying to build this with `./build.sh N200`, I am getting `libortp9` error. Also on Ubuntu 20.04, this package doesn't exists. Can someone please help me here?

**Comment by Mentors4EDU** [comment]
Referencing #95, but the fix notes say dev is only compatible w/ Ubuntu v16.04. Wanting to do a pull request but last time the code was updated was years ago...

**Comment by tom-2015** [comment]
Yes it's possible to get it working with current (Debian buster) versions (on Raspberry) but install the old libortp manually. For example download and install the .deb file from Stretch version manually: 
For AMD64: [http://ftp.de.debian.org/debian/pool/main/l/linphone/libortp-dev_3.6.1-3_amd64.deb](http://ftp.de.debian.org/debian/pool/main/l/linphone/libortp-dev_3.6.1-3_amd64.deb)
For Raspberry: [https://packages.debian.org/stretch/armhf/libortp-dev/download](https://packages.debian.org/stretch/armhf/libortp-dev/download)

You may also need to update the build.sh to remove the check for the library.

For building the Transceiver52M (needed of the new UHD driver and Ettus SDR devices) you need to compile with C++11 version. To do this type:
`export UHD_CFLAGS="-std=c++11"`
command before you start compilation.

You also need to change the UHDDevice.cpp file to support the new UHD logging uhd/utils/msg.hpp has been replaced by uhd/utils/log.hpp and uhd/utils/log_add.hpp. Check the Ettus documentation, some functions will need to be changed too.
